pkg/api/v1/testkube/model_test_suite_execution_extended.go

```diff
@@ -243,3 +243,12 @@ func (e *TestSuiteExecution) EscapeDots() *TestSuiteExecution {
 func (e *TestSuiteExecution) UnscapeDots() *TestSuiteExecution {
 	return e.convertDots(utils.UnescapeDots)
 }
+
```

Comment:
in fact, it's for TestSuite v2, for TestSuite v3 we need to clean BatchSteps

Reply:
Done